### ISSUE TITLE
Admin profile shown after login

### ISSUE THREAD

**Issue by cairoapcampos** [retspen/webvirtcloud#574]
By configuring WebVirtCloud in docker, I was able to add the compute node without any issues. However, after changing the admin user's password, this profile is shown every time after login.

What configuration should I do so that it is no longer shown after login?

I remove screen data because of the image being shared here.

![profile](https://user-images.githubusercontent.com/21186839/222560413-162c662f-7d07-46c6-9356-e06dd9941ff4.png)

**Comment by catborise** [comment]
yes it does. but because of link redirection. to fix refresh page. then enter full url like: http://localhost:8000

**Comment by cairoapcampos** [comment]
@catborise I'm trying to use WebVirtCloud in production. I configured SSL/TLS. SSL/TLS is defined on the Nginx reverse proxy. This way:

User access: https://webvirtcloud.mydomain.com -> Proxy redirect: http://IP:8085


**docker-compose.yml:**


- 8085:80


**nginx webvirtcloud.conf:**


proxy_pass http://IP:8085;


Access via domain name worked and I can access the VM consoles because port 443 was defined.

**No settings.py:**

```
# Websock port
WS_PORT = 6080

# Websock host
WS_HOST = "0.0.0.0"

# Websock public port - 80 or 443 if reverse-proxy, else 6080
WS_PUBLIC_PORT = 433

# Websock public host
WS_PUBLIC_HOST = None

# Websock public path
WS_PUBLIC_PATH = "/novncd/"

# Websock Certificate for SSL
WS_CERT = None

SOCKETIO_PORT = 6081
SOCKETIO_HOST = '0.0.0.0'

# Socketio public host
SOCKETIO_PUBLIC_HOST = None

# Socketio public port - 80 or 443 if reverse-proxy, else 6081
SOCKETIO_PUBLIC_PORT = 6081
```

I don't know where to set port 8000.

**Comment by catborise** [comment]
sorry for late answer. can you please add 

LOGIN_REDIRECT_URL="/instances/"

to settings.py file and restart webvirtcloud app
